Cargo.toml

```diff
@@ -18,6 +18,14 @@ travis-ci = { repository = "jaemk/cached", branch = "master" }
 default = ["proc_macro"]
 proc_macro = ["async-mutex", "cached_proc_macro"]
 
+[dependencies.hashbrown]
+version = "0.9"
+default-features = false
+features = ["raw"]
+# https://github.com/rust-lang/hashbrown/pull/202
+git = "https://github.com/cuviper/hashbrown"
+branch = "safer-raw-access"
```

Comment:
I made this a draft pending publication of https://github.com/rust-lang/hashbrown/pull/202.

We could do without that by using the existing `unsafe` methods, but I didn't want to be the first to introduce `unsafe` in this crate. :slightly_smiling_face:

Reply:
nice! Yes, I'd prefer if we could avoid using unsafe - but I'm open to it if that PR doesn't get approved

Reply:
OK, this is resolved with hashbrown 0.9.1!